src/tokens/ashib.ts

```diff
@@ -0,0 +1,9 @@
+import { defaultFetcherOptions, SupplyFetcher } from "../types";
+
+const ASHIB = "afc910d7a306d20c12903979d4935ae4307241d03245743548e767834153484942";
+
+const fetcher: SupplyFetcher = async (options = defaultFetcherOptions) => {
+    const total = 109000000000;
```

Comment:
You are missing the return of this function

Reply:
i guess then let's re-add "export default fetcher;".

Reply:
Everything appears to exist that needs to exist. If i am missing the return of a function, then i don't know where. everything looks fine to me.